osx/README.md

```diff
@@ -48,6 +47,30 @@ There are two ways to build the installer, automated or manually.
     It defaults to the `develop` branch at "https://github.com/learningequality/ka-lite/archive/develop.zip".
 
 
+## Changes on installer
+ 
+ * Responsible for setting up environment variable `KALITE_PYTHON`.
+ * Symlink for `Kalite` executable will be done during installation.
+ * Unpack assessment items.
+ * Bundle `KA-Lite.pkg` and `KA-Lite_Uninstall.tool` in `/Applications/KA-Lite` directory.
+
```

Comment:
Can you mention: We now save the ka-lite resources and pyrun in "/Users/Shared/ka-lite" or something similar?